python/ray/serve/controller.py

```diff
@@ -267,20 +269,31 @@ async def _recover_from_checkpoint(self, checkpoint_bytes):
         # Push configuration state to the router.
         # TODO(edoakes): should we make this a pull-only model for simplicity?
         for endpoint, traffic_policy in self.traffic_policies.items():
-            await self.router.set_traffic.remote(endpoint, traffic_policy)
+            await self.broadcast_routers(
+                lambda router: router.set_traffic.remote(
+                    endpoint, traffic_policy
+                )
+            )
```

Comment:
This isn't any more concise than (arguably even more verbose):

```
for router in self.routers:
    await router.set_traffic.remote(...)
```

Given that it's also less clear, I'd prefer to remove `broadcast_routers`